### ISSUE TITLE
Configure an ESLint rule for prefer export default

### ISSUE THREAD

**Issue by anniegiang** [FightPandemics/FightPandemics#213]
When there is only a single export from a module, prefer using default export over named export.

```
// bad.js

// There is only a single module export and it's a named export.
export const foo = 'foo';

```

```
// good1.js


export const foo = 'foo';

const bar = 'bar';

export default 'bar'; // There is a default export.

```

For your reference or inspiration:
https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/prefer-default-export.md

**Comment by manualzuru** [comment]
@robinv85 should we move this to closed?

**Comment by robinv85** [comment]
@manualzuru it's not fixed yet, we have ESLint configuration but if we change the rules then we have to update all the code so this is probably not a good time right now when a lot of development is happening, it will cause a lot of merge conflicts

**Comment by joshmorel** [comment]
I'm going to close this. It's kind of stale & we need to fix #786 first putting effort into adding more rules.
